operator/config/samples/loggingplumber_v1alpha1_flowtest.yaml

```diff
@@ -22,26 +22,26 @@ spec:
     - "[2021-06-10T11:50:08Z] @INFO Amo ideoque die se at, caro aer, ad cor. 1375 ::0.263548"
     - "[2021-06-10T11:50:09Z] @INFO Se contexo servis inpiis erogo, diligit ita significaret eosdem. 1376 ::0.405282"
 ---
-apiVersion: loggingplumber.isala.me/v1alpha1
-kind: FlowTest
-metadata:
-  name: clusterflowtest-sample
-  labels:
-    app.kubernetes.io/name: pod-simulation
-    app.kubernetes.io/managed-by: rancher-logging-explorer
-    app.kubernetes.io/created-by: logging-plumber
-    loggingplumber.isala.me/flowtest: clusterflowtest-sample
-spec:
-  referencePod:
-    kind: Pod
-    name: busybox-echo
-    namespace: default
-  referenceFlow:
-    kind: ClusterFlow
-    name: cluster-busybox-echo
-    namespace: "cattle-logging-system"
-  sentMessages:
-    - "[2021-06-10T11:50:06Z] @DEBUG Tam ipsae consuetudo infelix adtendi contexo mansuefecisti diutius re. 1373 ::0.403911"
-    - "[2021-06-10T11:50:07Z] @WARNING Ne hi flagitantur alienam neglecta. 1374 ::0.474177"
-    - "[2021-06-10T11:50:08Z] @INFO Amo ideoque die se at, caro aer, ad cor. 1375 ::0.263548"
-    - "[2021-06-10T11:50:09Z] @INFO Se contexo servis inpiis erogo, diligit ita significaret eosdem. 1376 ::0.405282"
+#apiVersion: loggingplumber.isala.me/v1alpha1
```

Comment:
we can just delete this.  With version control there is no need to keep commented out code.

Reply:
should I just uncomment this? I commented on this because there is a conflict with config-map using the same name. but it got fixed in #19